utils.js

```diff
@@ -25,6 +25,7 @@ export async function DiscordRequest(endpoint, options) {
     headers: {
       Authorization: `Bot ${process.env.DISCORD_TOKEN}`,
       'Content-Type': 'application/json; charset=UTF-8',
+      'User-Agent': 'Mozilla/5.0 (compatible; Discordbot/2.0; +https://discordapp.com)',
```

Comment:
This is invalid. The user agent must be `Discordbot ($url, $version)` as per the docs.

Reply:
mb

Reply:
do you have a 'example' from $version?